cmd/experimental/kjobctl/README.md

```diff
@@ -0,0 +1,50 @@
+# kjobctl
+ML/AI/Batch Jobs Made Easy
+
+## Description
+// TODO(user): An in-depth paragraph about your project and overview of use
+
+## Getting Started
+
+### Prerequisites
+- go version v1.22.0+
+- kubectl version v1.11.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
+
+### To Install
```

Comment:
This installs the API, but it doesn't "install" the binary. Maybe add an extra note about building the binary.

But this could be also left for when the binary actually does something.

Reply:
Added